src/components/AuthoritiesLookup/AuthoritiesLookup.js

```diff
@@ -117,6 +112,7 @@ const AuthoritiesLookup = ({
   const renderResultList = () => (
     <Pane
       id="authority-search-results-pane"
+      className={classNames(css.pane, css.focusIndicator, showDetailView ? css.hidden : undefined)}
```

Comment:
I think you can use `[css.hidden]: showDetailView` syntax instead of ternary. It's a part of `classnames` library